lib/IAMClient.js

```diff
@@ -345,6 +345,48 @@ class IAMClient {
     }
 
     /**
+     * Get policy evaluation (without authentication first)
+     * @param {Object} requestContextParams - parameters needed to construct
+     * requestContext in Vault
+     * @param {Object} requestContextParams.constantParams -
+     * params that have the
+     * same value for each requestContext to be constructed in Vault
+     * @param {[Object]} requestContextParams.paramaterize - params that have
```

Comment:
It is actually an optional object.  I meant to put the brackets around the name instead of the type to indicate it is optional.